src/main/kotlin/org/jitsi/nlj/transform/node/PcapWriter.kt

```diff
@@ -86,4 +90,14 @@ class PcapWriter(
 
         writer.dump(eth)
     }
+
+    fun close() {
+        if (lazyHandle.isInitialized() && writer.isOpen) {
```

Comment:
I think this should be checking `if (lazyWriter.isInitialized())`?

Reply:
done